### PR TITLE
Restore rollout after the change to point to appRevision

### PR DESCRIPTION
**Pull request by ryanzhang-oss**

### CODE REVIEW

File: charts/vela-core/values.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ image:
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
+useAppConfig: true
```
Comment (wonderflow) [inline charts/vela-core/values.yaml:19]: cool

File: charts/vela-core/templates/kubevela-controller.yaml [hunk 2]
```diff
@@ -112,6 +112,9 @@ spec:
             - "--webhook-port={{ .Values.webhookService.port }}"
             - "--webhook-cert-dir={{ .Values.admissionWebhooks.certificate.mountPath }}"
             {{ end }}
+            {{ if not .Values.useAppConfig }}
+            - "--app-config-installed=false"
+            {{ end }}
```
Comment (wonderflow) [inline charts/vela-core/templates/kubevela-controller.yaml:117]: can we add a conditional statement for the crd ?

**Review by wonderflow — APPROVED** [review]
LGTM, the CI seems still failing

File: charts/vela-core/templates/kubevela-controller.yaml (hunk 2 above)
Comment (ryanzhang-oss) [reply charts/vela-core/templates/kubevela-controller.yaml:117]: is there a way to not install the appConfig CRD?

Comment (wonderflow) [reply charts/vela-core/templates/kubevela-controller.yaml:117]: I'm not sure, maybe we need create two charts

Comment (wonderflow) [reply charts/vela-core/templates/kubevela-controller.yaml:117]: One is application only(name is `kubevela`), one is application with AppConfig(name is `kubevela-compatible`).

**Pull request merged**